example/src/pages/ContentTracking.tsx

```diff
@@ -56,7 +56,7 @@ const ContentTrackingPage = () => {
         <h1>{pageData.title}</h1>
         <p>{pageData.description}</p>
         <h1>Import</h1>
-        <code>{`import { ContentTracking } from '@piwikpro/gatsby-piwik-pro';`}</code>
+        <code>{`import { ContentTracking } from '@piwikpro/gatsby-plugin-piwik-pro';`}</code>
```

Comment:
Please check out readme or npm page. There is a mismatch in naming

Reply:
What's wrong?